pkg/input/arm/functions_string.go

```diff
@@ -0,0 +1,97 @@
+// © 2022-2023 Snyk Limited All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package arm
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/vincent-petithory/dataurl"
+)
+
+// Functions from https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-string#base64
+
+// TODO delete me
+func fnImpl(e *EvaluationContext, args ...interface{}) (interface{}, error) {
```

Comment:
Remove this?

Reply:
oof good catch, that was copy-paste fodder that made it in. How embarassing!